src/pheval/cli_pheval.py

```diff
@@ -50,16 +52,18 @@
     required=False,
     help="The path of the configuration file (optional e.g config.yaml)",
 )
-def run(input_dir, testdata_dir, runner, tmp_dir, output_dir, config) -> None:
+def run(
+    input_dir: Path, testdata_dir: Path, runner: str, tmp_dir: Path, output_dir: Path, config: Path
+) -> None:
     """PhEval Runner Command Line Interface
 
     Args:
-        input_dir (Click.Path): The input directory (relative path: e.g exomiser-13.11)
```

Comment:
Can you remind me what prompted this change?

Reply:
This is outdated :)